include/psp2/json.h

```diff
@@ -973,7 +1050,7 @@ class Parser
 	 * 					  current character.
 	 * @param[in]  data - Data to pass to the callback function
 	 */
-	static int parse(Value& val, ParseCallback cb, void* data);
+	static int parse(Value& val, ParseCallBack cb, void* data);
```

Comment:
I preper `Callback` than `CallBack`; now a day, it's an one word.
so how about change the `db.yml`?

Reply:
Made the change in db.yml.